src/events.js

```diff
@@ -96,7 +95,7 @@ function dispatchMoveEvents({state, event}, hook, elements, checkElements) {
 
 function handleClickEvents(state, event, options) {
   const listeners = state.listeners;
-  const elements = getElements(state, event, options.interaction);
+  const elements = getElements(state.visibleElements, event, options.interaction);
```

Comment:
isn't this a breaking change if we only use visible elements now while normally we used all elements?

Reply:
No, it isn't.

the `state.visibleElements` was invoked previously in the function mode (see changes below):

```javascript
function filterElements(state, event, options) {
  return state.visibleElements.filter
```